packages/tocco-util/src/notifier/notificationActionFactory.js

```diff
@@ -61,16 +60,12 @@ export function getBlockingInfo(id, title, message, icon) {
   const options = {
     timeOut: 0,
     showCloseButton: false,
-    transitionIn: 'bounceInDown',
-    transitionOut: 'bounceOutUp',
     attention: true,
     onAttentionClick: () => {},
     component: () => <TitleMessage title={title} message={message}/>
   }
 
-  if (icon) {
-    options.icon = <Icon icon={icon} size="2x" />
-  }
+  options.icon = _isString(icon) ? <Icon icon={`${icon}`} size="3x" /> : <Icon icon="exclamation-triangle" size="3x" />
```

Comment:
why do you use a template strings for icon?

and why are you using a ternary operator like this in the first place and not:

  options.icon =<Icon icon={ _isString(icon) ? icon : 'excelmation-triangle'} size="3x" />